tests/Benchmarks/System/RangePerf.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+
+namespace System.Benchmarks
```

Comment:
Please follow the namespace and directory structure of other benchmarks.
For instance see this:
https://github.com/dotnet/corefxlab/blob/67620c5a10103a72c5125cc85ac123e56943ed60/tests/Benchmarks/System.Text.Primitives/AsciiDecoding.cs#L8

This should be within the System.Buffers.Experiemental.Benchmarks namespace and be placed within the tests/Benchmarks/System.Buffers.Experiemental folder.

Also consider renaming the file to exclude words like Perf/Benchmark/etc as they are redundant. RangeEnumeration works.

Reply:
> Please follow the namespace

I thought I did, but went with the type's namespace (i.e. `System`) instead.

> Also consider renaming the file to exclude words like Perf/Benchmark/etc as they are redundant. RangeEnumeration works.

I though that's the convention after seeing `Utf8StringPerf`, `JsonReaderPerf`, `JsonWriterPerf`. etc.

I fixed both issues.

Reply:
> I though that's the convention after seeing `Utf8StringPerf`, `JsonReaderPerf`, `JsonWriterPerf`. etc.

Great point. It looks like we don't have a consistent rule on this. Thanks for fixing it.